package.json

```diff
@@ -69,7 +69,8 @@
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "lefthook": "*"
```

Comment:
Is this required? Can't we remove `lefthook` dependency completely?

Reply:
Good point, i don't think `users` need it, as long as it is part of `devDependencies` we are good.